### ISSUE TITLE
Make NPM/Bower module

### ISSUE THREAD

**Issue by batje** [Dominique92/Leaflet.GeoJSON.Ajax#7]
That would make it a bit easier for other users to find and use the module.

**Comment by Dominique92** [comment]
Good idea. That can be valuable for some. Feel free to create a GeoJSON.Ajax npm as yours.
Don't hesitate to modify / add any necessary file accordingly in the GIT
